.dockerignore

```diff
@@ -0,0 +1,8 @@
+LICENSE
+README.md
+test/*
+.argconfig
+.dockerignore
+.idea
+.git
+.gitignore
```

Comment:
Also add node_modules/, definitely the most important directory to ignore.

Reply:
added